internal/web3ext/web3ext.go

```diff
@@ -108,6 +108,12 @@ web3._extend({
 			call: 'clique_status',
 			params: 0
 		}),
+		new web3._extend.Method({
+			name: 'getSigner',
+			call: 'clique_getSigner',
+			params: 1,
+			inputFormatter: [web3._extend.formatters.inputBlockNumberFormatter]
```

Comment:
```suggestion
			inputFormatter: [null]
```
Otherwise it will surely reject rlp hexdata, no?

Reply:
Weirdly it did not reject rlp hexdata, but I removed it anyway